tests/test_bug.py

```diff
@@ -13,7 +13,6 @@ def test_get_bug_labels():
     assert classes[1087488]
     assert not classes[1101825]
     # labels from regression_bug_nobug.csv
-    assert not classes[447581]  # nobug
```

Comment:
Could you get another one from regression_bug_nobug.csv that is "nobug"?

Reply:
I searched yesterday, and none of them overlaps with bug IDs in `classes`. We have only 37 bugs in `classes`.

Reply:
They're probably the bugs from the mock file. We might have to add a new one then as the `test_trainer` test is failing too

Reply:
Done in fb18f14c737c34d39e4340df4ad47f94bc67e18c